README.md

```diff
@@ -17,6 +17,29 @@ It requires python 3.6 or above, as well as the modules:
 In order to process ``.acq`` files, it needs [`bioread`](https://github.com/uwmadison-chm/bioread), an excellent module
 that can be found at [this link](https://github.com/uwmadison-chm/bioread).
 
+Linux and mac installation:
+Donwload the package as zip from github and uncompress or if you have ``git`` use the command:
+
+`` git clone https://github.com/smoia/phys2bids.git``
+
+open a terminal in the phy2bids folder and execute the command:
+
+``sudo python3 setup.py install``
```

Comment:
```suggestion
``python setup.py develop``
```
Can you add a line about python3 and the possibility of needing sudo (without it being the default)?